webrender_traits/src/lib.rs

```diff
@@ -27,10 +25,6 @@ extern crate core_graphics;
 #[cfg(target_os = "windows")]
 extern crate dwrote;
 
-#[cfg(feature = "serde_codegen")]
-include!(concat!(env!("OUT_DIR"), "/types.rs"));
-
-#[cfg(feature = "serde_derive")]
 include!("types.rs");
```

Comment:
Could we just inline this file here or use `mod`? This will allow better error messages.